testing/content/files/en-us/web/images/bad_src/index.md

```diff
@@ -4,13 +4,13 @@
 ---
 
 <!-- exists on disk but isn't actually a real SVG file. -->
-<img src="actuallynota.svg">
+![](actuallynota.svg)
 
 <!-- resolves, intially, to a directory -->
-<img src="/en-us/web">
+![](/en-us/web)
 
 <!-- actually isn't a valid PNG -->
-<img src="actuallynota.png">
+![](actuallynota.png)
 
 <!-- is a zero byte image -->
-<img src="empty.gif">
+![](empty.gif)
```

Comment:
Let's keep one of the HTML images (if necessary, wrapping it in a table), to be able to test that we check Markdown and HTML images.